src/core/src/signature.rs

```diff
@@ -534,6 +535,39 @@ impl Signature {
         None
     }
 
+    // return single corresponding sketch
+    pub fn get_sketch(&self) -> Option<&Sketch> {
+        if self.signatures.len() != 1 {
+            if self.signatures.len() > 1 {
+                dbg!("Multiple sketches found! Please run select first.");
```

Comment:
either do a `todo!` here (to panic), or change the function signature to return an error? I think `todo!` is fine by now, but since `dbg!` only prints a message we can get weird behavior that will be hard to debug later...
```suggestion
                todo!("Multiple sketches found! Please run select first.");
```